notebooks/404-style-transfer-webcam/README.md

```diff
@@ -0,0 +1,24 @@
+# Live Style Transfer with OpenVINO™
+
```

Comment:
We need to start with an explanation of this notebook. It is confusing starting with "Binder is a free..."

Reply:
Updated with the description of the notebook instead of binder.

Reply:
Love the adds